task/deprecated-image-check/0.1/README.md

```diff
@@ -0,0 +1,22 @@
+# deprecated-image-check task
+
+## Description:
+A check for deprecated images that are no longer maintained and will accumulate security issues.
```

Comment:
Will this check accumulate security issues?

Reply:
I found this description a little confusing. Does the phrase "will accumulate security issues":
* is a result of the fact that images are no longer maintained, or
* is related to this task in a way that this task will list down (accumulate) detected security issues?